Code/GraphMol/Depictor/RDDepictor.cpp

```diff
@@ -451,57 +450,160 @@ unsigned int compute2DCoordsMimicDistMat(
   return cid;
 }
 
+namespace detail {
+class MatchScorer {
```

Comment:
There's no documentation for either what the class does or how it does it.
I'm pretty sure I get it, but if you're going to add something like this, please document it somewhere.

Reply:
Good point, I had completely forgotten about that helper class. I have moved it to `SubstructUtils.cpp` and exposed two functions that use that class in `SubstructUtils.h` as they may useful to others.
I would also like exposing them to Python but as there is no `rdSubstructMatch` module I am not sure where would be the best module to host them (`rdchem`? Suggestions very welcome).